src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -1168,10 +965,10 @@ maybe_update_index_file(State) ->
     end.
 
 update_index_file(State) ->
```

Comment:
Suggest upgrading the logging here to `notice` level and including more details about the index file that we're attempting to update.